src/applications/check-in/utils/validateVeteran/index.js

```diff
@@ -105,14 +103,14 @@ const validateLogin = async (
       goToNextPage();
     }
   } catch (e) {
+    // @TODO simplify this logic
     setIsLoading(false);
     if (e?.errors[0]?.status !== '401' || isMaxValidateAttempts) {
-      let params = '';
```

Comment:
this should get added back but with may something like `let errorType = 'lorota-fail'`

something like this 
```if (e?.errors[0]?.status !== '401' || isMaxValidateAttempts) {
      let errorType = 'lorota-fail'
      if (e?.errors[0]?.status === '410' || isMaxValidateAttempts) {
        errorType = 'max-validation';
      }
      updateError(errorType);```

Reply:
then remove the `else {
        updateError('validation');
      }`

Reply:
I think `lorota-fail` is better because its not describing a validation issue, its an issue with the api call. All validation errors will be the 410.